docs/UserManuals/ConfigUI/webhook.md

```diff
@@ -4,16 +4,16 @@ sidebar_position: 7
 description: Config UI instruction for Webhook
 ---
 
-Visit config-ui: `http://localhost:4000`.
-### Add and Update  Data Connections
+Visit config-ui: `http://localhost:4000` if you deployed via docker-compose.
+
+### Add a new webhook
 ![image](https://user-images.githubusercontent.com/3294100/191309840-460fbc9c-15a1-4b12-a510-9ed5ccd8f2b0.png)
 
-#### Connection Name
-Name your connection.
+#### Webhook name
 We recommand that you give your webhook connection a unique name so that you can identify and manage where you have used it later.
 
-### Use Webhook
-After cicking on "Generate POST URL", you will find four webhook URLs. Copy the ones that suits your usage into your CI or issue tracking systems. You can always come back to the webhook page to copy the URLs later on.
+### Use Webhooks
+After clicking on "Generate POST URL", you will find four webhook URLs. Copy the ones that suits your usage into your CI or issue tracking systems. You can always come back to the webhook page to copy the URLs later on.
 
 ![image](https://user-images.githubusercontent.com/3294100/191400110-327c153f-b236-47e3-88cc-85bf8fcae310.png)
```

Comment:
Not a doc issue, but rather the wording of the [message|https://user-images.githubusercontent.com/3294100/191400110-327c153f-b236-47e3-88cc-85bf8fcae310.png] displayed in the UI:
"Copy the following URLs to your issue tracking tool for Incidents and CI tool for Deployments by making a POST to DevLake."
It sounds to me that we're asking the user to make a POST to Devlake, whereas we're wanting to say the CI/Tracking tool will need to make the POST call, right? I think the wording can be improved there. Thoughts @hezyin @e2corporation ?